lib/sb3.js

```diff
@@ -125,6 +125,12 @@ const extensions = function (list) {
     };
 };
 
+const metadata = function (meta) {
+    return {
+        origin: meta.origin ? meta.origin : ''
```

Comment:
This will ensure all sb3s have an origin field, whether it be empty or not. This would be different from sb2s that don't include the key at all. Is that fine?

Reply:
Yeah, I think that makes sense, especially if vm isn't setting it. Changed the code!